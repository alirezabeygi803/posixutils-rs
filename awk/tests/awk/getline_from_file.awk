BEGIN {
	getline record < "tests/awk/test_data.txt";
	print record;
	getline < "tests/awk/test_data.txt";
	print $0;
	print NF;
	print $1;
	print $2;
	print $3;
	print $4;
	close("tests/awk/test_data.txt");
	getline < "tests/awk/test_data.txt";
	print $0;
	print NF;
	close("tests/awk/test_data.txt");
}

{
	getline record < "tests/awk/test_data.txt";
	print record;
	print $0;
	print NF, NR, FNR;
	getline < "tests/awk/test_data.txt";
}

{
	print $0;
	print NF, NR, FNR;
}
