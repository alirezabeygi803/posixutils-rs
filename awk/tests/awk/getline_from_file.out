1 Jane janitor 30
2 Smith sailor 45
4
2
Smith
sailor
45
1 Jane janitor 30
4
1 Jane janitor 30
1 Jane janitor 30
4 1 1
2 Smith sailor 45
4 1 1
3 Roselyn radiologist 28
2 Smith sailor 45
4 2 2
4 Megan machinist 32
4 2 2
5 Celine clergy 87
3 Roselyn radiologist 28
4 3 3
6 Fredrick farmer 19
4 3 3
7 Benjamin programmer 53
4 Megan machinist 32
4 4 4
4 Megan machinist 32
4 4 4
7 Benjamin programmer 53
5 Celine clergy 87
4 5 5
5 Celine clergy 87
4 5 5
7 Benjamin programmer 53
6 Fredrick farmer 19
4 6 6
6 Fredrick farmer 19
4 6 6
7 Benjamin programmer 53
7 Benjamin programmer 53
4 7 7
7 Benjamin programmer 53
4 7 7
